pipeline/pipeline/backend.py

```diff
@@ -162,7 +163,13 @@ class BatchBackend(Backend):
     """
 
     def __init__(self, url):
-        self._batch_client = batch.client.BatchClient(url)
+        self.session = aiohttp.ClientSession(
```

Comment:
Does this need to be an instance attribute? Not used elsewhere

Reply:
fixed